sdk/python/kfp/dsl/pipeline_config.py

```diff
@@ -63,6 +80,10 @@ def get_workspace(self) -> dict:
     def set_size(self, size: str):
```

Comment:
I think to prevent `size` from being set directly (e.g. `config.size = "invalid"`). We may have to have getter method for size leveraging the `@property` concept.

Reply:
Or perhaps converting `set_size` to a setter method and then have `set_size` just assign it directly